### DISCUSSION TITLE
Error when displaying a Mapbox map in superset

### DISCUSSION THREAD

**Discussion by galemoine** [apache/superset#22400] [Q&A / Help]
When I try to display any Mapbox based chart (MapBox, deck.gl Scatterplot, deck.gl 3D Hexagon etc...) I get a javascript error from mapbox-gl.js
My points are well displayed but not the map background.
MAPBOX_API_KEY is however well valued in the configuration file.

#### How to reproduce the bug

1. clone the master repository
2. add the MAPBOX_API_KEY in the configuration file under /superset/docker/pythonpath_dev/superset_config_docker.py
3. add the MAPBOX_API_KEY to .env-non-dev file
4. launch docker-compose -f docker-compose-non-dev.yml up -d
5. on the deck.gl Demo dashboard the elements are present but the background map is not displayed

### Expected results

The MapBox chart is well diplayed with a background map

### Actual results

The elements are well displayed but not the map background.
Some javascript errors are present in the console on mapbox-gl.js, it's all the same : Error: t.split is not a function

#### Screenshots

![image](https://user-images.githubusercontent.com/3499177/206741757-50351e59-2590-4998-a0e2-3021abae3cdb.png)

### Environment

- browser type and version: Tested with firefox and opera
- superset version: 2.0 (master)
- python version: 3.8.10
- node.js version: v16.13.1
- any feature flags active: EMBEDDED_SUPERSET

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

I work on a company network but I also did the test on my personal pc on my own network and I have exactly the same behavior.
I also tried to install superset via pip instead of Docker but I still get the same error

**Comment by rusackas** [comment]
Hi there! Just to confirm, were you following the steps outlined [here](https://github.com/apache/superset/tree/master/docker#readme) which involve steps including cloning the full example config (not just the mapbox key)?

I believe this is a configuration issue, and not a bug (numerous other orgs would have reported this if it were a bug, I believe), so I'm converting this to a Q&A discussion. You're also welcome to follow up on Slack if it helps in the #debug-together channel. If indeed there's a regression with mapbox key parsing, we'll certainly re-open this issue and continue pursuing it!

**Reply by galemoine** [reply]
Hi !

Curiously after a docker-compose -f docker-compose-no-dev.yml pull I have no more errors.
I must have made the clone of the repository master at the wrong time :)

Sorry for the inconvenience.

**Reply by rusackas** [reply]
Cool! I'll delete this thread then, but please feel free to spin up a new one if that's ever needed!
